newsfragments/2743.feature

```diff
@@ -0,0 +1 @@
+``dials.cosym``: Add dimension optimisation for intensity-based dataset clustering
```

Comment:
This should be dials.correlation_matrix, as we are only making this available through correlation_matrix.

Reply:
True - have made this change :)